bin/search_solr_tools

```diff
@@ -12,6 +13,48 @@ class SolrHarvestCLI < Thor
     puts SearchSolrTools::VERSION
   end
 
+  desc 'errcode', 'Get a description of the specified error exit code.  Leave empty to see all codes'
+  def errcode(code = -1)
```

Comment:
Need to document the optional parameter.

Reply:
Will refactor `desc` (following the Thor webpage examples) to better document this.